code/game/machinery/status_display.dm

```diff
@@ -80,7 +89,8 @@ var/global/list/status_displays = list() //This list contains both normal status
 		spookymode = 0
 		remove_display()
 		return
-	update()
+	if(mode != MODE_SHUTTLE_TIMER && mode != MODE_CARGO_TIMER) // handled in their subsystems
```

Comment:
I'll put this here, but the other subsystems do not catch the power outage case, or the spookymode case. This proc DOES, but it does so every 2 seconds. So, in a power outage or ghost situation, the displays will behave oddly, depending on which subsystem Byond decides to run first. The displays will be on more than intended in any case.

You could move the if(mode) checks to the top of this proc (flipping the logic so that if shuttle or cargo, return) and copy the other power/ghost checks into the 1 SECOND subsystems... Kinda ugly though.

Alternatively, a whole new subsystem (1 second) specifically for status screens, removing them from the machinery subsystem. Keeps everything clean and localized here.

Reply:
found another way to do it